arm-insights/2015-05-01/swagger/ApplicationInsights_SyntheticMonitor.json

```diff
@@ -0,0 +1,558 @@
+          "description": "User defined name if this WebTest."
+        },
+        "Description": {
+          "type": "string",
+          "description": "Purpose/user defined descriptive test for this WebTest."
+        },
+        "Enabled": {
+          "type": "boolean",
+          "description": "Is the test actively being monitored, default is false."
+        },
+        "Frequency": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Interval between test runs for this WebTest."
+        },
+        "Timeout": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Seconds until this WebTest will timeout and fail."
+        },
+        "Kind": {
+          "x-ms-client-name": "WebTestKind",
+          "type": "string",
+          "description": "The kind of web test this is, valid choices are ping and multistep.",
+          "default": "ping",
+          "enum": [
+            "ping",
+            "multistep"
+          ],
+          "x-ms-enum": {
+            "name": "WebTestKind",
+            "modelAsString": false
+          }
+        },
+        "RetryEnabled": {
+          "type": "boolean",
+          "description": "Allow for retries should this WebTest fail."
+        },
+        "Locations": {
+          "type": "array",
+          "description": "A list of where to physically run the tests from to give global coverage for accessibility of your application.",
+          "items": {
+            "$ref": "#/definitions/WebTestGeolocation"
+          }
+        },
+        "Configuration": {
+          "type": "object",
+          "description": "An XML configuration specification for a WebTest.",
+          "properties": {
+            "WebTest": {
+              "type": "string",
+              "description": "The XML specification of a WebTest to run against an application."
+            }
+          }
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Current state of this component, whether or not is has been provisioned within the resource group it is defined. Users cannot change this value but are able to read from it. Values will include Succeeded, Deploying, Canceled, and Failed.",
+          "readOnly": true
+        }
+      }
+    },
+    "WebTestGeolocation": {
+      "description": "Geo-physical location to run a web test from.",
+      "type": "object",
+      "properties": {
+        "Id": {
+          "type": "string",
+          "description": "Unique ID of this WebTest."
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The Azure subscription Id."
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client API version."
```

Comment:
All documents of composite Swagger need to agree about parameters defined on the global scope 100%. In this case, AutoRest complains about a mismatching description ;-) In another document in here I see (note capitalization):
```
    "ApiVersionParameter": {
      "name": "api-version",
      "in": "query",
      "required": true,
      "type": "string",
      "description": "Client Api Version."
    },
```
Please make sure all documents use precisely the same `ApiVersionParameter` description.